frontend/src/modules/products/components/ProductRow.tsx

```diff
@@ -46,9 +45,6 @@ export const ProductRow: React.FC<ProductRowProps> = ({ product }) => {
         <Link href={`/products/${product.id}`}>{product.name}</Link>
       </Td>
       <Td isNumeric>{product.price}</Td>
-      <Td>
-        <Checkbox defaultChecked={product.bulk} disabled />
-      </Td>
```

Comment:
here as well

Reply:
we agreed that creating a product is bulk by default thus, we dont need that checkbox